src/manager/lotus.rs

```diff
@@ -145,6 +145,27 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
         }
         Ok(map)
     }
+
+    async fn fund(&self, _subnet: SubnetID, _from: Address, _amount: TokenAmount) -> Result<()> {
+        todo!()
+    }
+
+    async fn release(&self, _subnet: SubnetID, _from: Address) -> Result<()> {
+        todo!()
+    }
+
+    async fn propagate(&self, _subnet: SubnetID, _from: Address, _postbox_cid: Cid) -> Result<()> {
+        todo!()
+    }
+
+    async fn whitelist_propagator(
+        &self,
+        _subnet: SubnetID,
```

Comment:
We need to specify the cid of the message to add the propagator to, right?

Reply:
updated and resolving this.